src/main/java/xyz/kmahyyg/eshopdemo/controller/GoodController.java

```diff
@@ -0,0 +1,60 @@
+package xyz.kmahyyg.eshopdemo.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import xyz.kmahyyg.eshopdemo.dao.SysItemsDao;
+import xyz.kmahyyg.eshopdemo.dao.SysUsersDao;
+import xyz.kmahyyg.eshopdemo.model.SysItems;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+public class GoodController {
+    @Autowired
+    private SysItemsDao sysItemsDao;
+
+    @RequestMapping("show/good/detail")
```

Comment:
How about change `good` to `item`? We must make sure everybody in our group share the same term about the same thing?

However, it depends on you.